### DISCUSSION TITLE
Dialog Cards - Paragraph Format

### DISCUSSION THREAD

**Discussion by emmanuelpineda** [Lumieducation/H5P-Nodejs-library#2331] [Q&A] [answered]
Is there any way to add more options to the paragraph format menu for the Dialgo Cards content type?

I can only choose one option as shown in the following image:

![image](https://user-images.githubusercontent.com/45979790/171334191-0bffbc2f-7fbd-45c8-a41e-a25460903acc.png)

Thank you in Advance.

**Comment by emmanuelpineda** [comment]
I found the solution by editing the file: semantics.json of the library 

![image](https://user-images.githubusercontent.com/45979790/171340120-8140ef62-0acc-4cd2-b041-3a100dc7c224.png)

**Comment by sr258** [comment, marked as answer]
It's better to use the in-built hook to alter semantics than to directly modify library files. The files will be overwritten once someone installs an upgrade from the H5P Hub or by uploading a h5p package with a newer version.

Check out the docs here: https://docs.lumi.education/advanced-usage/customization#changing-the-semantics-of-individual-libraries
